linter.py

```diff
@@ -25,13 +26,13 @@ class Flow(Linter):
     version_requirement = '>= 0.1.0'
     regex = r'''(?xi)
         # Warning location and optional title for the message
-        ^.+/(?P<file_name>[^/]+\.(js|html)):(?P<line>\d+):(?P<col>\d+),\d+:\s*(?P<message_title>.+)$\r?\n
+        ^.+\/(?P<file_name_1>.+):(?P<col_1>(\d+:\d+,(\d+:)?\d+)):\s(?P<message_title>.+)?\r?\n
 
-        # Main lint message
-        ^(?P<message>.+)$
+        # (Optional) main message
+        (^(?P<message>.+))?
```

Comment:
Is there ever a case where the main message is missing? It wouldn't be very useful to highlight a line and not explain what is wrong with it.